gulpfile.js

```diff
@@ -153,31 +179,18 @@ gulp.task('test-react-warnings', function() {
     gutil.log(gutil.colors.red.bold(message));
   };
 
-  return new IndexFileStream(require('./lib/index-static.jsx'))
+  return createIndexFileStream()
     .on('end', function() {
       console.warn = oldWarn;
       if (warnings) {
         this.emit('error', new Error('At least one warning was logged.'));
       }
     })
-    .pipe(gulp.dest('./dist'));
+    .on('data', function() {});
```

Comment:
I was reading https://nodejs.org/api/stream.html#stream_class_stream_readable yesterday but got confused cuz for things like https://github.com/mozilla/teach.webmaker.org/pull/839/files#diff-b9e12334e9eafd8341a6107dd98510c9R229 we are not adding the `on data` event listener there.  But yea, things will get a lot more clear to me after our Vidyo chat :smile: